requirements.txt

```diff
@@ -13,3 +13,8 @@ ethereum-serpent
 repoze.lru
 gevent-websocket==0.9.4
 cachetools>=2.0.0,<3.0.0
+sphinx
```

Comment:
Sphinx is already given as a requirement in `requirements-dev.txt` and so is `bumpversion`.

We have the developer and user requirements separated, please remove them from here. This is also why the tests fail from the get-go.